docs/guides/querying.md

```diff
@@ -2,195 +2,221 @@
 
 ## Starting the server
 
-Compass interfaces with an elasticsearch cluster. Run compass using:
+Compass interfaces with a postgres and elasticsearch cluster. Run compass using:
```

Comment:
should we update `installation.md` scope to also include
- migration
- serving
?

Reason is, so that each feature that we want to create the guide of does not need to include those again. wdyt?

Reply:
yes make sense